drivers/gles3/shader_compiler_gles3.cpp

```diff
@@ -241,11 +241,13 @@ void ShaderCompilerGLES3::_dump_function_deps(SL::ShaderNode *p_node, const Stri
 		r_to_add += "\n";
 
 		String header;
+		// cppcheck-suppress nullPointerDefaultArg
```

Comment:
This is again a situation where I should explain things better. Above these lines is a macro invocation `ERR_FAIL_COND(!fnode);` to ensure fnode is not null. cppcheck isn't smart enough to understand that this macro call is doing a null check.